### ISSUE TITLE
Feature request: Simple API for lines+markers in Plotly express

### ISSUE THREAD

**Issue by malmaud** [plotly/plotly.py#2719]
Thanks for the great work on Plotly express!

I think it would be nice to have a concise `express` function to plot a line chart with markers shown. This can be done now with

```
fig=express.line(...)
fig.update_traces(mode='markers+lines') 
```

but that seems quite verbose for a fairly common kind of figure. What about, for example, adding a keyword argument to `express.line` to specify marker properties? 

cf matplotlib, where you can conveniently do this with something like `plt.plot(x, y, fmt='.-')` to get line plot with markers.

**Comment by nicolaskruchten** [comment]
I've also found it annoying to reach for `update_traces` just to turn markers on! I don't think we'll do something as terse as `fmt` as shown but we should do something, indeed.

I'm torn between a single `markers=True` or a more full-blow `symbol="column"` type option. With the latter, just turning markers on would be `symbol=px.Constant(True)` or something like that.

**Comment by nicolaskruchten** [comment]
> I'm torn between a single `markers=True` or a more full-blow `symbol="column"` type option.

I implemented both in #3326 :)
